WordPress/Classes/Services/BloggingPrompts/BloggingPromptsService.swift

```diff
@@ -182,10 +185,11 @@ class BloggingPromptsService {
     ///   - blog: When supplied, the service will perform blogging prompts requests for this specified blog.
     ///     Otherwise, this falls back to the default account's primary blog.
```

Comment:
Nitpick: Now that we have `api` in the signature, this documentation should be updated.

If I read the code correctly, do you think this could do?

```suggestion
    ///     Otherwise, this falls back to the default account's primary blog.
    ///   - api: When supplied, the WordPressComRestApi instance to use to fetch the prompts.
    ///     Otherwise, an default or anonymous instance will be computed based on whether there is an account available.
```

Reply:
Thanks for catching that! 👍🏼